src/dlgTriggerEditor.cpp

```diff
@@ -6993,7 +6993,10 @@ void dlgTriggerEditor::slot_profileSaveAsAction()
                                                     tr("trigger files (*.trigger *.xml)"));
 
     if(fileName.isEmpty()) return;
-    fileName.append(".xml");
+    if (!fileName.endsWith(".xml"))
```

Comment:
You probably want to cast filename to lower case first to handle anyone who decides to capitalize xml